packages/zcli-core/src/lib/auth.test.ts

```diff
@@ -35,24 +31,13 @@ describe('Auth', () => {
         ZENDESK_EMAIL: 'test@zendesk.com',
         ZENDESK_API_TOKEN: 'test_api_token'
       })
-      .stub(auth, 'createBasicAuthToken', mockCreateBasicAuthToken)
       .it('should return basic token if ZENDESK_EMAIL and ZENDESK_API_TOKEN is set', async () => {
-        expect(await auth.getAuthorizationToken()).to.equal('Basic test@zendesk.com/token_test_api_token_base64')
-      })
-
-    test
-      .env({
-        ZENDESK_EMAIL: 'test@zendesk.com',
-        ZENDESK_PASSWORD: '123456'
-      })
-      .stub(auth, 'createBasicAuthToken', mockCreateBasicAuthToken)
-      .it('should return basic token if ZENDESK_EMAIL and ZENDESK_PASSWORD is set', async () => {
-        expect(await auth.getAuthorizationToken()).to.equal('Basic test@zendesk.com_123456_base64')
+        expect(await auth.getAuthorizationToken()).to.equal('Basic dGVzdEB6ZW5kZXNrLmNvbS90b2tlbjp0ZXN0X2FwaV90b2tlbg==')
```

Comment:
Just curious, where is the token 'dGVzdEB6ZW5kZXNrLmNvbS90b2tlbjp0ZXN0X2FwaV90b2tlbg==' coming from?

Reply:
Oh, that is the actual `base64` output from `createBasicAuthToken` function. I find the stubbing of `createBasicAuthToken` unnecessary since the function is predictable and no I/O calls, under our full control.